pom.xml

```diff
@@ -120,7 +121,7 @@
                     <rulesets>
                         <ruleset>/rulesets/java/braces.xml</ruleset>
                         <ruleset>/rulesets/java/strictexception.xml</ruleset>
-                        <ruleset>/rulesets/java/basic.xml</ruleset>
+                        <!--<ruleset>/rulesets/java/basic.xml</ruleset>-->
```

Comment:
I was getting the following error if I include that line.
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-pmd-plugin:3.5:check (test) on project dg-core: You have 1 PMD violation.